azure_arc_servers_jumpstart/privatelink/ARM/azuredeploy.json

```
@@ -0,0 +1,281 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "sqlAdminLogin": {
+            "type": "string",
+            "defaultValue": "sqladmin",
+            "metadata": {
+                "description": "Admin login for the SQL Server."
+            }
+        },
+        "sqlAdminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Password for the SQL Server."
+            }
+        },
+        "cloudVNetName": {
+            "type": "string",
+            "defaultValue": "Azure-NET",
```

```suggestion
            "defaultValue": "Azure-NET",
```
Azure Network? Azure VNET?